src/ui/patch_hub/patch_hub_core.sh

```diff
@@ -159,6 +159,16 @@ function show_registered_mailing_lists()
   message_box='Below, you can see the lore.kernel.org mailing lists that you have registered.'$'\n'
   message_box+='Select a mailing list to see the latest patchsets sent to it.'
 
+  fallback_message='kw could not find any registered mailing list.'$'\n'$'\n'
+  fallback_message+='Try to register mailing lists in the menu "Settings" then ' 
+  fallback_message+='"Register/Unregister Mailing Lists"'
```

Comment:
As this fallback message will one be used inside the `if` statement block, move these three lines to inside the `if` statemet block.